osquery/core/init.cpp

```diff
@@ -261,6 +262,16 @@ Initializer::Initializer(int& argc, char**& argv, ToolType tool)
       if (Flag::isDefault("database_path")) {
         osquery::FLAGS_database_path = homedir + "/shell.db";
       }
+      if (Flag::isDefault("getfile_path")) {
+        auto getfiledir = homedir + "/getfiles";
```

Comment:
Let's be safe, but not platform independent and use: [`mkdtemp`](http://pubs.opengroup.org/onlinepubs/9699919799/functions/mkdtemp.html).